gulpfile.js

```diff
@@ -527,6 +528,79 @@ function buildExperiments(options) {
 
 
 /**
+ * Build "Login Done" page.
```

Comment:
I can't yet - there's task somewhere in @erwinmombay 's queue to make it possible with compilation targets.